tests/integration/genutil/gentx_test.go

```diff
@@ -7,10 +7,12 @@ import (
 	"testing"
 	"time"
 
-	"github.com/stretchr/testify/suite"
+	"github.com/stretchr/testify/require"
 	tmproto "github.com/tendermint/tendermint/proto/tendermint/types"
+	"gotest.tools/v3/assert"
```

Comment:
Can we use any of `assert` or `require`? or both are required?

Reply:
Can't we write our own util using `assert`?

```go
func AssertNotPanics(fn func())  {
  defer {
    // recover and use assert to ensure no error
  }()
}
```

Reply:
certainly, I could do that in next PR's and will do a follow-up for genutil tests